### PR TITLE
Add example of how to manipulate timestamp on a write operation

### PR DESCRIPTION
**Pull request by kjeldflarup**
Signed-off-by: Kjeld Flarup <kfa@deif.com>

### CODE REVIEW

File: examples/server_settimestamp.c [hunk 1]
```diff
@@ -0,0 +1,94 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information. */
+
+/**
+ * Faking source timestamp
+ * ----------------------------
+ *
+ * This example shows how to write data with a modified source timestamp
+ * When showing the data in UAExpert, the source timestamp will always be half an hour being the server timestamp
```
Comment (ccvca) [inline examples/server_settimestamp.c:9]: Spelling error?
"be half an hour **being** the server timestamp"
being -> behind

**Comment by ccvca** [comment]
What do you think about setting also a custom StatusCode, so the example shows all features of setting a DataValue instead of a Variant?

File: examples/server_settimestamp.c (hunk 1 above)
Comment (kjeldflarup) [reply examples/server_settimestamp.c]: fixed.

**Comment by kjeldflarup** [comment]
@ccvca then the example should be called server_writedatavalue.c  
Would'nt that however be very alike this example https://github.com/open62541/open62541/blob/791eb95986709e0fdaeff6d64ea5b8f994198b55/examples/tutorial_server_variable.c#L98

**Comment by ccvca** [comment]
@kjeldflarup So it's not required here. (I didn't know tutorial_server_variable in detail before)

**Comment by kjeldflarup** [comment]
@ccvca A complete example is still a good idea. Like a tutorial.  
But I just had a need to make a proff of concept on the timestamp, and thought that I would share it. It is open for anybody to make the example "complete".

**Comment by jpfr** [comment]
Looking good!
Thanks for the contribution.

**Pull request merged**